src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaDataResultSets.java

```diff
@@ -1754,4 +1755,319 @@ static ImmutableList<JdbcColumnMetaData> buildImportedKeysColumnMetaData(
+                            JdbcType.BOOLEAN.name(), //type.name,
+                            boolean.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "FIXED_PREC_SCALE", //label,
+                            "FIXED_PREC_SCALE", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.BOOLEAN, //type.id,
+                            JdbcType.BOOLEAN.name(), //type.name,
+                            boolean.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "AUTO_INCREMENT", //label,
+                            "AUTO_INCREMENT", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.BOOLEAN, //type.id,
+                            JdbcType.BOOLEAN.name(), //type.name,
+                            boolean.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNullable, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "LOCAL_TYPE_NAME", //label,
+                            "LOCAL_TYPE_NAME", //columnName,
+                            null, //schemaName,
+                            0, //precision,
+                            0, //scale,
+                            Types.VARCHAR, //type.id,
+                            JdbcType.VARCHAR.name(), //type.name,
+                            String.class.getName()) //columnClassName
+                    )
+                    .add(new JdbcColumnMetaData(
+                            ordinal++,
+                            false, //caseSensitive,
+                            ResultSetMetaData.columnNoNulls, //nullable,
+                            true, //signed,
+                            64, //displaySize,
+                            "MINIMUM_SCALE", //label,
+                            "MINIMUM_SCALE", //columnName,
+                            null, //schemaName,
```

Comment:
this file exceeds 2000 lines, should it be split?

Reply:
I added an exception, because I don't know how to split this file. It doesn't change often.